modular_skyrat/modules/death_consequences_perk/death_consequences.dm

```diff
@@ -0,0 +1,73 @@
+
+	add_verb(human_holder, TYPE_VERB_REF(/mob, adjust_degradation))
+	add_verb(human_holder, TYPE_VERB_REF(/mob, refresh_death_consequences))
+
+	to_chat(human_holder, span_danger("You suffer from [DEATH_CONSEQUENCES_QUIRK_NAME]. By default, you will \
+	degrade every time you die, and recover very slowly while alive. This may be expedited by resting, sleeping, being buckled \
+	to something cozy, or using rezadone. \n\
+	As your degradation rises, so too will negative effects, such as stamina damage or a worsened crit threshold. \n\
+	You can alter your degradation on the fly via the Adjust resonance degradation verb, and change your settings via the Refresh death consequence variables verb."))
+
+/datum/quirk/death_consequences/remove()
+	var/mob/living/carbon/human/human_holder = quirk_holder
+	human_holder.cure_trauma_type(/datum/brain_trauma/severe/death_consequences, TRAUMA_RESILIENCE_ABSOLUTE)
+
+	remove_verb(human_holder, TYPE_VERB_REF(/mob, adjust_degradation))
+	remove_verb(human_holder, TYPE_VERB_REF(/mob, refresh_death_consequences))
+
+/mob/verb/adjust_degradation(increment as num)
+	set name = "Adjust resonance degradation"
+	set category = "IC"
+	set instant = TRUE
+
+	var/datum/brain_trauma/severe/death_consequences/linked_trauma = get_death_consequences_trauma()
+	var/mob/living/carbon/trauma_holder = linked_trauma?.owner
+	if (isnull(linked_trauma) || isnull(trauma_holder) || trauma_holder != mind.current) // sanity
+		to_chat(usr, span_warning("You don't have a body with death consequences!"))
+		return
+
+	if (!isnum(increment))
+		to_chat(usr, span_warning("You can artificially change the current level of your resonance degradation with this verb. \
+		You can use this to cause degradation in ways the customization cannot. <b>You need to enter a number to use this verb.</b>"))
+		return
+
+	if (linked_trauma.permakill_if_at_max_degradation && ((linked_trauma.current_degradation + increment) >= linked_trauma.max_degradation))
+		if (tgui_alert(usr, "This will put you over/at your maximum degradation threshold and PERMANENTLY KILL YOU!!! Are you SURE you want to do this?", "WARNING", list("Yes", "No"), timeout = 7 SECONDS) != "Yes")
+			return
+
+	linked_trauma.adjust_degradation(increment)
+	to_chat(usr, span_notice("Degradation successfully adjusted!"))
+
+/mob/verb/refresh_death_consequences()
+	set name = "Refresh death consequence variables"
+	set category = "IC"
+	set instant = TRUE
+
+	var/datum/brain_trauma/severe/death_consequences/linked_trauma = get_death_consequences_trauma()
+	var/mob/living/carbon/trauma_holder = linked_trauma?.owner
+	if (isnull(linked_trauma) || isnull(trauma_holder) || trauma_holder != mind.current) // sanity
+		to_chat(usr, span_warning("You don't have a body with death consequences!"))
+		return
+
+	linked_trauma.update_variables()
+	to_chat(usr, span_notice("Variables successfully updated!"))
+
+/mob/proc/get_death_consequences_trauma()
```

Comment:
Document this proc.

Also, why is this on the mob level, and not on `/mob/living/carbon/human`?

Reply:
because im a dingus

Reply:
aw wait no, i forgot, its because the verbs that use this can be used as a ghost, that was probably critical information i really shouldve included and im sorry for not

Reply:
the rationale is this: i just dont see a reason why being dead should prevent you from modifying how your degradation works, since after all, its still your body and these verbs are basically OOC tweaks to it anyway
maybe it should be in OOC tab in that case

Reply:
> the rationale is this: i just dont see a reason why being dead should prevent you from modifying how your degradation works, since after all, its still your body and these verbs are basically OOC tweaks to it anyway maybe it should be in OOC tab in that case

Yeah, that makes sense.